### PR TITLE
[BI-34] Updated UI for forgot password

### PR DESCRIPTION
**Pull request by alexstefan-binarcode**
I have updated the button with our BaseButton component and added a loading image when the loading state is true.

### CODE REVIEW

File: src/views/ForgotPassword.vue [hunk 1]
```diff
@@ -19,25 +19,38 @@
...
-            </button>
+              {{ $t('general.submit') }}
+            </base-button>
           </form>
         </ValidationObserver>
+        <img
+          v-if="loading"
+          class="h-5 w-5 mx-auto my-3"
+          src="https://icon-library.com/images/loading-icon-transparent-background/loading-icon-transparent-background-12.jpg"
+        >
```
Comment (alexstefan-binarcode) [inline src/views/ForgotPassword.vue:39]: I should remove this after the BaseButton version of @LAlex14 will be merged (the one with the loading image inside the button)

File: src/views/ForgotPassword.vue [hunk 2]
```diff
@@ -19,25 +19,38 @@
...
+      </div>
       </div>
     </div>
-  </div>
 </template>
 
 <script>
 import authService from '@/api/authService';
+import BaseButton from '@/components/BaseButton';
```
Comment (alexstefan-binarcode) [inline src/views/ForgotPassword.vue:48]: Also I should remove this if the BaseButton component will be used globally

**Review by alexstefan-binarcode — COMMENTED** [review]
Waiting for other reviews

**Review by Ceciliaa — APPROVED** [review]

File: src/views/ForgotPassword.vue [hunk 3]
```diff
@@ -19,18 +19,27 @@
...
 
-            <button
+            <router-link class="font-medium text-indigo-600 hover:text-indigo-500 text-sm" to="/login">
+              {{ $t('app.routerTitle.login') }}
+            </router-link>
+
+            <base-button
                 type="submit"
                 class="flex justify-center w-full px-4 py-2 my-4 text-sm font-medium text-white bg-indigo-600 border border-transparent rounded-md shadow-sm hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
                 :disabled="!valid"
```
Comment (LAlex14) [inline src/views/ForgotPassword.vue:29]: add :loading="loading"

File: src/views/ForgotPassword.vue [hunk 4]
```diff
@@ -19,18 +19,27 @@
...
-            </button>
+              {{ $t('general.submit') }}
+            </base-button>
           </form>
         </ValidationObserver>
+        <img
+          v-if="loading"
+          class="h-5 w-5 mx-auto my-3"
+          src="https://icon-library.com/images/loading-icon-transparent-background/loading-icon-transparent-background-12.jpg"
+        >
```
Comment (LAlex14) [inline src/views/ForgotPassword.vue:39]: delete this

**Review by LAlex14 — APPROVED** [review]

**Pull request merged**